### ISSUE TITLE
using `slim.batch_norm` get low accuracy about 0.1

### ISSUE THREAD

**Issue by IamMegatron2025** [soloice/mnist-bn#7]
hello, when I use `slim.batch_norm` as your `mnist_bn.py` did,  I get low accuracy about 0.1. But when I  use `tf.layers.batch_normalization` like:

```
    with slim.arg_scope([slim.conv2d, slim.fully_connected],
                        activation_fn=tf.nn.relu,
                        normalizer_fn=tf.layers.batch_normalization,
                        normalizer_params={'training': is_training, 'momentum': 0.95}):
```

the accuracy is about 0.98.. 
can you explain this situation？

**Comment by soloice** [comment]
I'm not pretty sure. But as for now, I think the recommended way to use BN in TF is `tf.layers.batch_normalization`. `slim` is too high level so that one can hardly understand what happened inside without diving into the source codes.
